src/Plugin/ConceptSubset/ContactLanguages.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_contact_forms\Plugin\ConceptSubset;
+
+use Drupal\Core\Entity\Query\QueryInterface;
+use Drupal\rdf_skos\ConceptSubsetPluginBase;
+
+/**
+ * Creates a subset of the languages vocabulary.
+ *
+ * @ConceptSubset(
+ *   id = "contact_languages",
+ *   label = @Translation("Contact Languages"),
+ *   description = @Translation("Languages to be used for on the contact form."),
+ *   concept_schemes = {
+ *     "http://publications.europa.eu/resource/authority/language"
+ *   }
+ * )
+ */
+class ContactLanguages extends ConceptSubsetPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function alterQuery(QueryInterface $query, $match_operator, array $concept_schemes = [], string $match = NULL): void {
+    $languages = [
```

Comment:
This should be tested against a fixture containing the languages it has to contain.